pom.xml

```diff
@@ -39,6 +39,11 @@
           <artifactId>pdfbox</artifactId>
           <version>2.0.11</version>
         </dependency>
+        <dependency>
+    		<groupId>org.apache.commons</groupId>
+    		<artifactId>commons-collections4</artifactId>
+    		<version>4.2</version>
+		</dependency>
```

Comment:
Please remove this. You are only using ListOrderedMap from it, but should probably use standard java.util.LinkedHashMap instead.